client-runtime/src/main/java/com/microsoft/rest/http/BufferedHttpResponse.java

```diff
@@ -16,49 +16,72 @@
  * HTTP response which contains already-buffered body content.
```

Comment:
Consider changing this to convey that body content is cached after being read once, not that it's already been buffered by virtue of being contained in this object.